src/BylawsText.js

```diff
@@ -0,0 +1,297 @@
+import React from 'react'
+import join from 'join-array'
+
+export default function BylawsText({ answers }) {
+  const answersArray = Object.entries(answers)
+  const anyAnswerGiven = answersArray.some(([_, answer]) => {
+    const isAnswerCheckbox = Array.isArray(answer)
+    return isAnswerCheckbox
+      ? answer.filter(Boolean).length > 0
+      : answer.length > 0
+  })
+
+  console.log('answers:', answers)
+
+  const nameAndLocationParagraph = (
+    <section id="nameAndLocation">
+      <h3>§ 1 Name und Sitz des Vereins</h3>
+      <p>
+        Der Name des Vereins lautet {answers.clubName} e.V. Der Verein hat
+        seinen Sitz in {answers.clubLocation}.
+      </p>
+    </section>
+  )
+
+  const businessYearParagraph = (
```

Comment:
You have a lot of components in this one file. I think it might make sense to put them into separate files. Maybe use an extra folder to keep this organised.

Reply:
I added an extra component named "Paragraph" in order to make the code easier to read :-)